web/styles/portico/footer.css

```diff
@@ -43,10 +43,12 @@
         text-transform: uppercase;
         border-bottom: 0;
         margin-bottom: 0;
+        margin-block-start: 0;
+        margin-block-end: 0;
```

Comment:
I can't tell what these lines do, visually; probably means it's OK but I haven't reviewed this.

Reply:
margin-bottom isn't working here because of default browser margin-block-start. We have to enforce 0 margin here.